dockerfiles/Dockerfile.dev

```diff
@@ -3,8 +3,11 @@ FROM    golang:1.8.3-alpine
 
 RUN     apk add -U git make bash coreutils
 
-RUN     go get github.com/LK4D4/vndr && \
-        cp /go/bin/vndr /usr/bin && \
+ARG     VNDR_COMMIT=9909bb2b8a0b7ea464527b376dc50389c90df587
+RUN     git clone https://github.com/LK4D4/vndr.git "/go/src/github.com/LK4D4/vndr" && \
+        cd "/go/src/github.com/LK4D4/vndr" && \
+        git checkout -q "$VNDR_COMMIT" && \
+        go build -v -o /usr/bin/vndr . && \
```

Comment:
> Isn't this going to break as soon as vndr adds a dependency, since we don't have a go get anywhere here?

I assume it would vendor that dependency, but it's a ood point; I took the same approach as is used in moby/moby, so we may want to modify the approach there as well